tool/tsh/common/tsh.go

```diff
@@ -968,6 +969,16 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	benchExecKube.Flag("container", "Selects the container to exec into.").StringVar(&benchKubeOpts.container)
 	benchExecKube.Flag("interactive", "Create interactive Kube session").BoolVar(&cf.BenchInteractive)
 
+	benchPostgres := bench.Command("postgres", "Run PostgreSQL database benchmark tests").Hidden()
+	benchPostgres.Flag("db-user", "Database user used to connect to the target database. The user must have enough permissions on the database to execute all the benchmark queries.").StringVar(&cf.DatabaseUser)
+	benchPostgres.Flag("db-name", "Database name where benchmark queries will be executed.").StringVar(&cf.DatabaseName)
+	benchPostgres.Arg("database", "Teleport target database name or the direct database URI. Available databases can be retrieved by running `tsh db ls`. When using direct connection, the benchmark will issue connections directly to this database, and no Teleport is involved in the testing. It must contain all the connection information, including authentication credentials.").StringVar(&cf.DatabaseService)
+
+	benchMySQL := bench.Command("mysql", "Run MySQL database benchmark tests").Hidden()
```

Comment:
Can we remove hidden command
```suggestion
	benchMySQL := bench.Command("mysql", "Run MySQL database benchmark tests")
```

so if hidden command name is know the help command is printed `tsh bench`  ?